charts/xui-mo-webapp/values.yaml

```diff
@@ -45,7 +45,9 @@ nodejs:
     S2S_SERVICE: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     FEE_AND_PAY_API: http://payment-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     SERVICES_TERMS_AND_CONDITIONS_API_SERVICE: http://xui-terms-and-conditions-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
-  keyVaults:
+    MANAGE_CASE_LINK: https://manage-case.platform.hmcts.net/cases
```

Comment:
The values.yaml file is used on AKS for all environmental variables, these values will be used on Production; they look fine to me, but just double check :)